CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [4.4.x]
```

Comment:
Lets combine [4.4.x] and [4.4.2] in a new [4.5.0]

Reply:
Updated